### ISSUE TITLE
UK deaths incorrect for June 1

### ISSUE THREAD

**Issue by greenwoodmark** [CSSEGISandData/COVID-19#2606]
https://coronavirus.data.gov.uk/  normally aligns with your data, but it has 
*111* for "Number of additional deaths on Monday 1 June 2020", while your deaths data published to 
`https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_deaths_global.csv`                   reports *556* new daily deaths (the 'deaths' field increases from 38489 on 2020-5-31 to 39045 on 2020-06-01)

**Comment by dmeadkagool** [comment]
@greenwoodmark just noticing this as well, did you ever figure out the reason?

**Comment by greenwoodmark** [comment]
No, I never did find out why there is this discrepancy. I also looked at
the Oxford Martin 'Our World in Data' repo but it has almost an identical
time series to the Johns Hopkins. In the end I closed the query and just
accepted it. It is annoying though that my model has failed to anticipate
the high daily deaths in the UK this past week. I reckon that the case
survival rate has dropped from 94% to 92% as a result.
